analyticsdataserver/settings/base.py

```diff
@@ -42,16 +42,31 @@
 
 ########## DATABASE CONFIGURATION
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#databases
+DEFAULT_MYSQL_OPTIONS = {
```

Comment:
Can you copy CACHES into here from local.py so that it will be defined?

Reply:
I have moved databases to local.py and copy caches from local.py to base.py